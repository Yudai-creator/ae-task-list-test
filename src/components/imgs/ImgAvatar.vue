<template>
    <img v-if="isClicked" width="30" :class="`${taskInput != '' ? 'opacity-100' : ' opacity-25'}`" src="https://ik.imagekit.io/u33i3sss0/figma_pfp(5)_Rrl76Qnuu.png?updatedAt=1692986988771" alt="me">
</template>

<script setup>

const props = defineProps({
    isClicked: Boolean,
    taskInput: String
})


</script>