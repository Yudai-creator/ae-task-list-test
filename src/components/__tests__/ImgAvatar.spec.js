import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import ImgAvatar from '../imgs/ImgAvatar.vue'

// describe('HelloWorld', () => {
//   it('renders properly', () => {
//     const wrapper = mount(HelloWorld, { props: { msg: 'Hello Vitest' } })
//     expect(wrapper.text()).toContain('Hello Vitest')
//   })
// })

describe('ImgAvatar', () => {
  const wrapper = mount(ImgAvatar)
  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('should have img', () => {
    expect(wrapper.find("img"))
  })
})

// describe('ImgAvatar', () => {
//   it('it has the image link', () => {
//     const wrapper = mount(ImgAvatar)
//     expect(wrapper.html()).toContain("https://ik.imagekit.io/u33i3sss0/figma_pfp(5)_Rrl76Qnuu.png?updatedAt=1692986988771")
//   })
// })