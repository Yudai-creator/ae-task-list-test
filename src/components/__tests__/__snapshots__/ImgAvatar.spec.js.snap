// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ImgAvatar > should render properly 1`] = `"<!--v-if-->"`;
